### ISSUE TITLE
Missing AWS regions

### ISSUE THREAD

**Issue by dxrpl** [Graylog2/graylog-plugin-aws#370]
HI there,

Is there any way to configure more regions? I need eu-north-1 at the moment but the list is going to grow since my application will run in at least three regions.

![Screenshot from 2020-05-21 15-28-41](https://user-images.githubusercontent.com/50060205/82564056-66206000-9b78-11ea-934e-e3e86924b587.png)

**Comment by radykal-com** [comment]
Hi,

What version of graylog are you running? I can see eu-north-1 in graylog 3.2.4
<img width="584" alt="Captura de pantalla 2020-05-23 a las 10 44 02" src="https://user-images.githubusercontent.com/2077281/82726124-62a8e800-9ce2-11ea-9e3b-2dc5e482398f.png">

**Comment by dxrpl** [comment]
I use 3.2 so this is probably the case. I'll upgrade to 3.3 and post back what happen.

**Comment by dxrpl** [comment]
OK now I see how much I'm out of reality.
I run Graylog via docker-compose and I need a few more plugins.
I just found this plugin in Graylog Marketplace https://marketplace.graylog.org/addons/3f132fab-50f0-4c88-b63d-9ac99aa6c20e

It's outdated quite a bit but without further reading I downloaded jar with version 2.4.5 and gone forward.

Next I mounted graylog/plugin in my local filesystem. That's the reason why:

1. after copying the jar into it I had outdated version without full AWS region support (duh)
2. when I throwed jar away there was no plugin installed by default (because while mounting volume its content is overwritten)

So... sorry guys for bothering you.
